experimenter/experimenter/experiments/tests/test_change_formatting_utils.py

```diff
@@ -315,6 +325,42 @@ def test_formatting_for_array_fields(self):
 
         self.assertDictEqual(change, expected_change)
 
+        field_name_segments = "segments"
+        field_diff_segments = {
+            "old_value": old_value_segments,
+            "new_value": new_value_segments,
+        }
+        change_timestamp_segments = self._create_formatted_timestamp(
+            comparison_log.changed_on
```

Comment:
I'm surprised that this works for both segments and outcomes. Maybe it's a strange side effect of using the same timestamp for both changelogs, that both changes get lumped together into one changelog? Is that what's happening here? I would have expected `comparison_log` to contain the info about the outcomes change but not the segments.